docs/src/snforge-advanced-features/storage-cheatcodes.md

```diff
@@ -160,4 +161,41 @@ fn store_in_complex_mapping() {
 > The `load` cheatcode will return zeros for memory you haven't written into yet (it is a default storage value for Starknet contracts' storage).
 
 
+## Example with `storage_address_from_base`
+
+This example use `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-contract-storage.html#addresses-of-storage-variables).
```

Comment:
```suggestion
This example uses `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-contract-storage.html#addresses-of-storage-variables).
```

Reply:
I would also mention that it requires importing appropriate `{field_name}ContractMemberStateTrait`

Reply:
Information about import added